fastlane/lanes/localization.rb

```diff
@@ -39,6 +39,7 @@
   { glotpress: 'el', android: 'el', promo_config: false },
   { glotpress: 'en-au', android: 'en-rAU', promo_config: false },
   { glotpress: 'en-ca', android: 'en-rCA', promo_config: false },
+  { glotpress: 'en-gb', android: 'en-rGB', promo_config: false },
```

Comment:
See details in message of commit 0ece140e74354a5dab14b14abcad2e024b355558

I accidentally removed the entire `en-gb` locale from the list during #17341, instead of _only_ removing the `en-gb => en-US` mapping from GlotPress to Play Store used for the Release Notes.

During this beta, when opening the PR, I then noticed that the entry for `en_GB` got removed from `available_languages.xml` (which is used in our Android code by our in-app locale picker), which led me to realize that we stopped updating `WordPress/src/values-en-rGB` and stopped downloading them from GlotPress, while this was not the intent. So I restored just the mapping for `glotpress` => `android` (used for `strings.xml`) in the list here, while making sure not to re-include `play_store: en-US` that was the cause of the bug and hack fixed by #17341 and the one we really wanted gone)

This means that the `21.0-rc-2` that went out to beta testers did not include the translations for `en-GB` (as I realized the issue, and added 0ece140e74354a5dab14b14abcad2e024b355558, only after the beta was build), and will only be re-included in the next beta or release.

Reply:
No dramas. It's a beta after all. Well done catching this.